src/SignalR/common/Protocols.Json/src/Protocol/JsonHubProtocol.cs

```diff
@@ -777,6 +777,10 @@ internal static JsonSerializerOptions CreateDefaultSerializerSettings()
             options.IgnoreNullValues = false;
             options.IgnoreReadOnlyProperties = false;
             options.PropertyNamingPolicy = JsonNamingPolicy.CamelCase;
+            options.PropertyNameCaseInsensitive = true;
```

Comment:
I believe MVC is leaving this as false. Should there be consistency between ASP.NET Core frameworks?

// @rynowak

Reply:
yeah, we were leaving this as false. I'm interested to know why folks think case-insensitive is a better option 😆

Reply:
I believe it's because Json.NET is true by default.

Reply:
Ryan is making a stand that Json.NET has the wrong default. I don't feel strongly either way other than SignalR and MVC should be consistent

*#TeamConsistency*

Reply:
I don't feel incredibly strongly about SignalR and MVC being consistent. I do feel strongly that case-insensitive is the wrong choice for building a REST api. 

I think it's a wierd idea to we would do case-insenstive because legacy.

Reply:
As long as `JsonNamingPolicy.CamelCase` affects reading (IIRC it didn't in Json.NET?) I'm OK with leaving this false.

Reply:
Looks like it affects reading. `{"someName":"value"}` will serialize into `class C { public string SomeName { get; set; }` with CamelCase set, and doesn't if it isn't.

Reply:
The policy effects serializing and deserializing.